xilem_core/src/view.rs

```diff
@@ -30,11 +37,16 @@ use crate::{message::MessageResult, DynMessage, Mut, ViewElement};
 /// During message handling, mutable access to the app state is given to view nodes,
 /// which will in turn generally expose it to callbacks.
 ///
+/// Due to restrictions of the [orphan rules](https://doc.rust-lang.org/reference/items/implementations.html#orphan-rules),
+/// `ViewMarker` needs to be implemented for every type that implements `View`, see [`ViewMarker`] for more details.
+///
```

Comment:
It might be worth repeating the same code block here as well

Reply:
Something like in the last commit?

Reply:
Yeah, exactly.